### PR TITLE
Rover: rename sonar to rangefinder (NFC)

### PR DESCRIPTION
**Pull request by khancyr**
To prepare rework for proximity

### CODE REVIEW

File: APMrover2/sensors.cpp [hunk 1]
```diff
@@ -127,67 +127,67 @@ void Rover::accel_cal_update() {
...
+            obstacle.rangefinder1_distance_cm < static_cast<uint16_t>(obstacle.rangefinder2_distance_cm))  {
             // we have an object on the left
             if (obstacle.detected_count < 127) {
                 obstacle.detected_count++;
             }
-            if (obstacle.detected_count == g.sonar_debounce) {
-                gcs().send_text(MAV_SEVERITY_INFO, "Sonar1 obstacle %u cm",
-                        static_cast<uint32_t>(obstacle.sonar1_distance_cm));
+            if (obstacle.detected_count == g.rangefinder_debounce) {
+                gcs().send_text(MAV_SEVERITY_INFO, "rangefinder1 obstacle %u cm",
```
Comment (OXINARF) [inline APMrover2/sensors.cpp]: Not very important, but maybe better to keep the initial letter uppercase? There's another one below.

Comment (khancyr) [inline APMrover2/sensors.cpp]: ho ! I miss those ! thx

**Comment by khancyr** [comment]
rebased and correction from review

**Comment by OXINARF** [comment]
I fixed one lower/uppercase that you missed and merged. I hope I didn't overstep but it looked an easy one. Thanks!

**Pull request closed without merging**